setup.py

```diff
@@ -52,9 +41,9 @@
     },
     entry_points={
         'console_scripts': [
-            "forte-medical-train=forte_medical_cli.train:main",
-            "forte-medical-process=forte_medical_cli.process:main",
-            "forte-medical-evaluate=forte_medical_cli.evaluate:main",
+            "forte-medical-train=ForteHealth_cli.train:main",
```

Comment:
I think the convention is `forte_health_cli`, but do we even have this class anywhere? We should remove these entry points if we don't even have them.